schemas/latest/landingzones/types/aks.json

```diff
@@ -15,17 +15,26 @@
           "title": "AKS"
       },
       "Value": {
-          "type": "object",
-          "additionalProperties": false,
-          "properties": {
-              "version": {
-                  "type": "string"
-              }
-          },
-          "required": [
-              "version"
-          ],
-          "title": "Value"
-      }
+        "type": "object",
```

Comment:
Can you update the changelog located in `schemas/latest/readme.md`

Reply:
Can you also parameterize these parameters?  They are hardcoded as defaults and would be better to move them out to config as part of this change.

- podCidr // default = '11.0.0.0/16'
- serviceCidr // default = '20.0.0.0/16'
- dnsServiceIP // default = '20.0.0.10'
- dockerBridgeCidr // default = '30.0.0.1/16'